google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/GapicSpannerRpc.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner.spi.v1;
+
+import com.google.cloud.spanner.SpannerException;
+import com.google.cloud.spanner.spi.v1.SpannerRpc.Option;
+import com.google.longrunning.Operation;
+import com.google.protobuf.FieldMask;
+import com.google.spanner.admin.database.v1.Database;
+import com.google.spanner.admin.instance.v1.Instance;
+import com.google.spanner.admin.instance.v1.InstanceConfig;
+import com.google.spanner.v1.BeginTransactionRequest;
+import com.google.spanner.v1.CommitRequest;
+import com.google.spanner.v1.CommitResponse;
+import com.google.spanner.v1.ExecuteSqlRequest;
+import com.google.spanner.v1.PartitionQueryRequest;
+import com.google.spanner.v1.PartitionReadRequest;
+import com.google.spanner.v1.PartitionResponse;
+import com.google.spanner.v1.ReadRequest;
+import com.google.spanner.v1.RollbackRequest;
+import com.google.spanner.v1.Session;
+import com.google.spanner.v1.Transaction;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+
+/** Implementation of Cloud Spanner remote calls using Gapic libraries. */
+public class GapicSpannerRpc implements SpannerRpc {
+
+  // private final DatabaseAdminClient adminDatabaseClient;
+  // private final InstanceAdminClient adminInstanceClient;
```

Comment:
Remove dead code.

Also, when we actually implement this, we should probably have this Rpc class use the stub classes instead of the client classes.